src/otaclient/create_standby/update_slot.py

```diff
@@ -247,5 +247,8 @@ def update_slot(self) -> None:
         self._process_non_regular_files()
         self._process_regular_file_entries()
 
+        if self._interrupted.is_set():
+            raise UpdateStandbySlotFailed("failure during regular files processing!")
```

Comment:
The code looks good. How about adding an E2E test case to ensure this exception is raised correctly?